ResearchKit/Common/ORKSurveyAnswerCellForScale.m

```diff
@@ -119,9 +119,7 @@ - (IBAction)sliderValueChanged:(id)sender {
 }
 
 + (CGFloat)suggestedCellHeightForView:(UIView *)view {
-    return 140.0;
+    return 244.0;
 }
 
```

Comment:
Yuan - not sure quite how that would work! This is a class method on the base class, and vertical/horizontal is very specific to this class. It might be easiest to just create a subclass of ORKSurveyAnswerCellForScale to cover the vertical case.